life-sciences/1.1/modules/ROOT/pages/index.adoc

```diff
@@ -1,5 +1,6 @@
 = MuleSoft Accelerator for Life Sciences
 :ls-version: {page-component-version}
+:page-aliases: @life-sciences::ls-assets.adoc
```

Comment:
Hey Tasha, I notice this is the only one that doesn't have a version number. Does it need one?

Reply:
Oh no, I just tested it and it's actually not working. I'm going to add the version and check again.